interactors/src/main/java/com/thefuntasty/interactors/disposables/SingleDisposablesOwner.kt

```diff
@@ -13,8 +13,40 @@ interface SingleDisposablesOwner {
         return execute(args, { })
     }
 
-    fun <ARGS, T> BaseSingler<ARGS, T>.execute(args: ARGS): Disposable {
-        return execute(args, { })
+    fun <T> BaseSingler<Unit, T>.execute(result: SinglerResult.Builder<T>.() -> Unit): Disposable {
```

Comment:
DRY The body of this function is implemented three times in this interface. You can call one function from another
```suggestion
    fun <T> BaseSingler<Unit, T>.execute(result: SinglerResult.Builder<T>.() -> Unit): Disposable {
        execute(Unit, result)
    }
```